arch/arm/core/cortex_m/cmse/arm_core_cmse.c

```diff
@@ -131,9 +131,43 @@ int arm_cmse_addr_nonsecure_read_ok(u32_t addr, int force_npriv)
 	return _arm_cmse_addr_nonsecure_read_write_ok(addr, force_npriv, 0);
 }
 
-int arm_cmse_addr_nonsecure_read_write_ok(u32_t addr, int force_npriv)
+int arm_cmse_addr_nonsecure_readwrite_ok(u32_t addr, int force_npriv)
 {
 	return _arm_cmse_addr_nonsecure_read_write_ok(addr, force_npriv, 1);
 }
 
+static int _arm_cmse_addr_range_nonsecure_read_write_ok(u32_t addr, u32_t size,
```

Comment:
Also, could we have similar utils for secure domain? It will be hard to write tests/self-checks without them.